apps/enc.c

```diff
@@ -613,7 +613,7 @@ static int set_hex(char *in, unsigned char *out, int size)
         *(in++) = '\0';
         if (j == 0)
             break;
-        if (!isxdigit(j)) {
+        if (!ossl_isxdigit(j)) {
```

Comment:
But here on the other hand it's appropriate to be locale-independent. Because code below kind of is. Or is it really? Note that OPENSSL_hexchar2int is actually locale-neutral and attempts to return an error code if it's given non-hex digit, but caller effectively ignores it. Maybe if would be more appropriate to *omit* isxdigit call and pay attention to return value from OPENSSL_hexchar2int?